docs/installing-northstar/troubleshooting.md

```diff
@@ -1,28 +1,42 @@
 # Troubleshooting
 
-Generally try to first launch the vanilla game (i.e. not Northstar) if you encounter any issue and see if it also occurs there as well. Some problems can occur when the vanilla game was never launched before using Northstar. 
+Generally try to first launch the vanilla game (i.e. not Northstar) if you encounter any issue and see if it also occurs there as well. Some problems can occur when the vanilla game was never launched before using Northstar.
 
 A lot of problems around the game failing to communicate with Origin can also be prevented by launching Origin before Northstar should you encounter any issues in that regard.
 
-Also note that some solutions described below can also apply to the base game, like issues caused by 10th+ gen Intel CPUs.
+Also note that some solutions described below can also apply to the base game, like issues caused by [10th+ gen Intel CPUs](#cant-join-servers-issues-with-10th-gen-intel-cpus).
```

Comment:
Link didn't work for me when testing on [your gitbook mirror](https://amione.gitbook.io/northstarwiki/installing-northstar/troubleshooting).